requirements/base.in

```diff
@@ -1 +1,3 @@
 open-api-framework
+
+git+https://github.com/maykinmedia/commonground-api-common@feature/update-notifs-client
```

Comment:
Once this is released, we should bump it in open-api-framework and then bump that instead. Open Klant doesn't seem to be affected by the zgw-consumers update, so let's keep this PR open and change it once the releases are done

Reply:
This also applies to https://github.com/open-zaak/open-zaak/pull/1800 and https://github.com/open-zaak/open-notificaties/pull/196 and the upcoming PR's for the other projects too.